.mypy.ini

```diff
@@ -0,0 +1,92 @@
+; This configuration file is temporary and will be moved to `pyproject.toml` when it is:
+; a) finished, or b) mypy supports cascading configuration and we can have the base
+; configuration in `pyproject.toml` and additional configuration here for WIP modules.
+
+[mypy]
+python_version = 3.11
+files = astropy/astropy/**/*.py
+plugins = numpy.typing.mypy_plugin
+show_error_codes = true
+# TODO: temporary
+ignore_missing_imports = true
+follow_imports = skip
```

Comment:
I've found `silent` to be more useful than `skip`. From what I can tell `silent` helps ensure that the annotations are self-consistent (although it cannot check if the annotations are consistent with unannotated code), but `skip` does not check self-consistency.